components/graph-line.vue

```diff
@@ -169,19 +172,20 @@ export default {
         return ['line', 'scatter', 'images'].indexOf(value) !== -1
       },
     },
+    parameterId: {
+      type: String,
+      default: '',
+    },
   },
-  data: () => ({
-    isCollapsed: false,
-  }),
   computed: {
     ...mapGetters('preferences/theme', ['colors']),
+    ...mapGetters('map', ['getCollapsedDatasets']),
+    isCollapsed() {
+      return this.getCollapsedDatasets.includes(this.parameterId)
+    },
   },
   methods: {
-    toggle() {
-      if (this.collapsible) {
-        this.isCollapsed = !this.isCollapsed
-      }
-    },
+    ...mapMutations('map', ['toggleCollapsedDataset']),
```

Comment:
What does this do?

Reply:
Added comments to toggleCollapsedDataset function. 

    // Updates the collapsedDatasets array, when id already exists in this Array
    // it will be removed from the array, otherwise it will be added.